src/drawing/polygon.rs

```diff
@@ -123,8 +113,10 @@ where
         );
     }
     let first = poly[0];
```

Comment:
```suggestion
    let first = poly.first().unwrap();
```

Reply:
Switching from indexing to first/last is more verbose and less idiomatic.